src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/jql/Jql.kt

```diff
@@ -0,0 +1,19 @@
+package com.atlassian.performance.tools.jiraactions.api.jql
+
+import com.atlassian.performance.tools.jiraactions.api.SeededRandom
+import com.atlassian.performance.tools.jiraactions.api.page.IssuePage
+
+interface JqlContext {
```

Comment:
Looks like and public SPI. Historically we have many issues with SPI written in Kotlin and how it interoperated with Java and Java interfaces and their default methods. I would suggest writhing this part in Java which should not be costly but may pay-off in case we want to extend the SPI in the future without breaking the compatibility.